### PR TITLE
Update bootstrap sdk to 9.0.200

### PR DESCRIPTION
**Pull request by kasperk81**
https://github.com/dotnet/msbuild/pull/10836#discussion_r1806780808

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,9 +1,11 @@
 {
   "sdk": {
+    "version": "9.0.200-preview.0.24518.5",
+    "rollForward": "major",
     "allowPrerelease": true
   },
   "tools": {
-    "dotnet": "9.0.100-rc.2.24474.11",
+    "dotnet": "9.0.200-preview.0.24518.5",
```
Comment (rainersigwald) [inline global.json:8]: I wouldn't update the toolset if we can avoid it, I'd rather not be on such an early prerelease.

File: eng/Version.Details.xml [hunk 2]
```diff
@@ -89,13 +89,13 @@
       <Uri>https://github.com/nuget/nuget.client</Uri>
       <Sha>19756345139c45de23bd196e9b4be01d48e84fdd</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.12.0-3.24510.2">
+    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.13.0-1.24517.9">
```
Comment (rainersigwald) [inline eng/Version.Details.xml:92]: @dotnet/kitten can you go over our darc subscriptions? we should have been on Roslyn 4.13 already.

**Comment by mmitche** [comment]
> @mmitche can we get https://dotnetbuilds.azureedge.net/public/Sdk/9.0.200-preview.0.24518.5/dotnet-sdk-9.0.200-preview.0.24518.5-osx-arm64.tar.gz uploaded to https://dotnetcli.azureedge.net/dotnet/Sdk/9.0.200-preview.0.24518.5/dotnet-sdk-9.0.200-preview.0.24518.5-osx-arm64.tar.gz? this is for msbuild bootstrap, the tool is sensitive to the text like `curl: (56) The requested URL returned error: 404` on stdout/stderr even if final exit code was successful.

That's the customer release point and can't be used for pre-release builds. We shojld figure out how to make it work with dotnetbuilds

File: eng/Version.Details.xml (hunk 2 above)
Comment (YuliiaKovalova) [reply eng/Version.Details.xml:92]: the subscription to roslyn vs17.13 wasn't created
fixed: https://github.com/dotnet/msbuild/pull/10852/files

Comment (YuliiaKovalova) [reply eng/Version.Details.xml:92]: feel free to pull the changes from https://github.com/dotnet/msbuild/pull/10852 

I will merge it later today.

**Comment by rainersigwald** [comment]
That's the version of the .NET runtime that's packaged into 9.0.200 builds right now: https://github.com/dotnet/sdk/blob/10e47c896095ad44c053e99d4e4664bd02984bab/eng/Version.Details.xml#L18

@marcpopMSFT, is there an easy way to flow the rc2 runtime to SDK release/9.0.200 so we can run bits built with rc2 on it?

**Comment by marcpopMSFT** [comment]
There is not. It has to be flowed manually. That had been prepped but it was lost when the 1xx to 2xx flow was redone last week.

**Comment by marcpopMSFT** [comment]
https://github.com/dotnet/sdk/pull/44364

**Review by surayya-MS — APPROVED** [review]

**Review by YuliiaKovalova — APPROVED** [review]
Thank you for contribution and fix in the task :)

**Pull request merged**